specifications/service-registry.yaml

```diff
@@ -0,0 +1,279 @@
+        the service catalog file.
+      operationId: announceCatalog
+      requestBody:
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              type: object
+              properties:
+                uri:
+                  description: Location of the service catalog resource to register
+                  type: string
+                  format: catalogUri
+              required:
+                - uri
+      responses:
+        '200':
+          description: Successful operation
+        '403':
+          description: Catalog announcement forbidden
+        '405':
+          description: Invalid input
+  /catalogs/list:
+    get:
+      tags:
+        - catalogs
+      summary: List registered service catalogs
+      description: 
+        List service catalog URIs registered with this service registry. The 
+        query can be used by other Service Registries to mirror the content of
+        another registry.
+      operationId: listCatalogs
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/catalog-listing'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/catalog-listing'
+        '403':
+          description: Catalog listing forbidden
+  /services/find:
+    get:
+      tags:
+        - queries
+      summary: Query registered services
+      description: Receives a list of services fulfilling the specified query criteria.
+      operationId: findServices
+      parameters:
+        - in: query
+          name: lat
+          description: latitude of the user location (WGS84)
+          schema:
+            type: number
+        - in: query
+          name: lon
+          description: longitude of the user location (WGS84)
+          schema:
+            type: number
+        - in: query
+          name: radius
+          description: Radius around the coordinates to search in kilometers
+          schema:
+            type: number
+        - in: query
+          name: certs
+          description:
+            Return only service providers owning the specified certificate
+            in their certificate chain
+          schema:
+            type: array
+            items:
+              type: string
+              format: uuid
+        - in: query
+          name: types
+          description: Return only services matching the specified types
+          style: form
+          explode: false
+          schema:
+            type: array
+            items:
+              type: string
+              enum:
+                - route-gtfs
+                - route-netex
+                - routing-otp
+                - taxi-suti
+                - tickets-lippu
+                - tickets-bob
+                - tickets-ticketapi
```

Comment:
As above, would make sense to add a `maas-api` case where for conformance with [the "MaaS API"](https://github.com/maas-alliance/apis/blob/master/specifications/maas-api.yaml#L64)

Reply:
Would also make sense to add a case for the [booking.yaml](https://github.com/maas-alliance/apis/blob/master/specifications/booking.yaml#L498), too

Reply:
I added maas-api and maas-booking to the API lists. Generally I thing those are also items that should be harmonized between all API specifications.